dlib/dnn/loss_abstract.h

```diff
@@ -431,6 +435,9 @@ namespace dlib
                 - 0 < min_target_size <= target_size
                 - 0.5 < min_detector_window_overlap_iou < 1
             ensures
+                - scale_invariant = true
+                - This function should be used when scale-invariance is desired, and an
```

Comment:
Here we should explicitly reference the input layers we expect the user to use.  So explicitly say they should be using the `input_rgb_image_pyramid`.

Reply:
Fixed.